corehq/apps/export/templates/export/customize_export_new.html

```diff
@@ -55,6 +55,18 @@
         <fieldset>
             <div class="form-group">
                 <label
+                    for="export-type"
+                    class="col-sm-3 col-md-2 control-label">
+                    {% if export_instance.type == 'case' %}{% trans "Case Type" %}{% else %}{% trans "Form" %}{% endif %}
```

Comment:
considered abstracting both the name and value into the classes, but it felt like more trouble than it was worth and not obviously cleaner